app/modules/settings/actions/settings-login-user.js

```diff
@@ -0,0 +1,15 @@
+import { settingsUpdateUserState } from "./settings-update-user-state"
+
+const ipcRenderer = require("electron").ipcRenderer
+
+export const settingsLoginUser = (assignmentURL) => {
+  return dispatch => {
+    return new Promise((resolve) => {
+      ipcRenderer.send("requestAuthorization", assignmentURL)
```

Comment:
Send request authorization to main process so that window is opened for authorization